Antidote/ChatViewController.m

```diff
@@ -373,4 +384,21 @@ - (void)updateLastReadDate
     [[AppContext sharedContext].profileManager.toxManager.objects changeChat:self.chat lastReadDateInterval:interval];
 }
 
+- (void)startCallButtonPressed
+{
+    OCTSubmanagerCalls *manager = [AppContext sharedContext].profileManager.toxManager.calls;
+
+    OCTCall *call = [manager callToChat:self.chat enableAudio:YES enableVideo:NO error:nil];
```

Comment:
If I do this, what's the best way to give the user an option to call with no audio? Or do you want them to have that selection in the `DialingCallViewController` before calling?